task-driver/Dockerfile

```diff
@@ -1,31 +1,38 @@
-# Used for running integration tests on a simulated MPC network
-FROM rust:1.63-slim-buster AS builder
+# Build a set of sources that are used to build a dependency cache for the main
+# integration tests. We effectively copy all the sources and then remove the target
+# directory. This means that changes to `task-driver` sources will invalidate the
+# cache for this stage only, as the end state is the same
+FROM alpine AS sources
+COPY . .
+RUN rm -rf task-driver
 
-# Create a build dir and add local dependencies
-WORKDIR /build
+# Add a stage for the contract sources, we pull compiled artifacts out of this stage
+FROM renegade-contracts as contract-sources
```

Comment:
my docker lingo is a bit rusty 😅 where is the `renegade-contracts` image getting pulled from? just wanna make sure it's using the old version of the contracts for now as you mentioned

Reply:
This gets built and cached locally in the `cargo-integrate` command